### PR TITLE
Various small cleanups.

### PR DESCRIPTION
**Pull request by freshp86**
Merge variables declaration and definitions.
Remove redundant check in elementIsScrollLocked().
Extract common functionality in _clearElementCaches.
Add braces around single-line if for consistency with the rest of this file.

### CODE REVIEW

**Comment by cdata** [comment]
Thanks for the cleanups!

**Comment by cdata** [comment]
Side note: we don't have a strict guideline for style today, but in the future it would be good to ask about subjective stylistic changes before implementing them.

File: iron-dropdown-scroll-manager.html [hunk 1]
```diff
@@ -136,6 +129,11 @@
         return this._unlockedElementCache.indexOf(element) > -1;
       },
 
+      _clearElementCaches: function() {
```
Comment (cdata) [inline iron-dropdown-scroll-manager.html]: This method should probably be `__private`

**Comment by valdrinkoshi** [comment]
Hi @freshp86, this looks outdated as we moved the manager into iron-overlay-behavior. If you're still facing issues please open an issue in https://github.com/PolymerElements/iron-overlay-behavior/issues/new 👌

**Pull request closed without merging**